index.js

```diff
@@ -13,6 +13,19 @@ export class UpdateAPK {
 
   get = (url, success, error, options = {}) => {
     fetch(url, options)
+      .then(response => {
+        if (!response.ok) {
+          let message;
+          if (response.statusText) {
+            message = `${response.url}  ${response.statusText}`
+          }
+          else {
+            message = `${response.url} Status Code:${response.status}`
+          }
+          throw Error(message);
```

Comment:
I believe this is potentially a breaking change yes? We will be throwing errors here where we did not use to throw errors and that is an API difference.

I am fine with that, I just want to ask so we may be clear and issue a semver-major version with a note in the CHANGELOG

Reply:
hi, actually this is not breaking change.
For example, the server returns 500 server internal errors
If we don't throw the error here, JSON.parse will be executed later and will still throw an error, unfortunately the JSON.parse error is thrown instead of the real cause of the error